terraform/modules/emr/templates/emr/get_scripts.sh

```diff
@@ -0,0 +1,30 @@
+SOURCE_LOCATION=$1
+DESTINATION_LOCATION=$2
+
+BUCKET="${config_bucket}"
+
+SOURCE_PATH=$BUCKET/$SOURCE_LOCATION
+
+(
+    # Import the logging functions
+    source /opt/emr/logging.sh
+
+    function log_wrapper_message() {
+        log_message "$${1}" "get_scripts.sh" "$${PID}" "$${@:2}" "Running as: ,$USER"
```

Comment:
Is this just passing $1 into the log_message function? Why are we not passing $2.. $5? (at least $2 should be INFO?